website/docs/r/vra_project.html.markdown

```diff
@@ -107,6 +107,8 @@ A project resource supports the following arguments:
 
 ## Argument Reference
 
+**Due to the design of IAAS API to update a project, it's not able to add and remove user or group at the same time. Please execute `terraform apply` twice.**
```

Comment:
Can we add an example scenario? Like what to expect in both the apply calls with a specific example